### ISSUE TITLE
Problems setting it up

### ISSUE THREAD

**Issue by marko-kisegi** [Nithanim/gw2api#25]
I tried setting it up new gradle command is apparently     compileOnly 'me.nithanim.gw2api:gw2api:0.0.8'
Then I tried making a Main class to which I got with the first command I tried running from the tutorial
Caused by: java.lang.ClassNotFoundException: me.nithanim.gw2api.v2.GuildWars2Api

**Comment by Nithanim** [comment]
Hello! I think for gradle you need to use `implementation` instead of `compileOnly`. `compileOnly` would mean not to include it for runtime, and therefore you get the `ClassNotFoundException` because it is not included. Source:https://stackoverflow.com/questions/61696863/gradle-compileonly-and-runtimeonly

My word of warning: The library is insanely old so it uses a lot of old dependencies and might be broken. I don't know. I started a rewrite a long time ago but never finished it. I hope it still works for you!

**Comment by marko-kisegi** [comment]
True. I was mistaken. Its not working well for me I'm just struggling but thanks!
